frontend/packages/kubevirt-plugin/src/components/form/form-row.tsx

```diff
@@ -39,30 +40,31 @@ export const FormRow: React.FC<FormRowProps> = ({
         type === ValidationErrorType.Info || type === ValidationErrorType.Warn ? message : undefined
       }
       className={className}
-    >
-      {help && (
-        <span className="kubevirt-form-row__icon-status-container">
-          <PopoverStatus
-            title={`${fieldId} help`}
-            hideHeader
+      labelIcon={
+        help && (
+          <Popover
+            position={PopoverPosition.right}
+            aria-label={`${fieldId} help`}
             onHide={onHide}
             onShow={onShow}
-            statusBody={
-              <StatusIconAndText
-                title={`${fieldId} help`}
-                iconOnly
-                icon={
-                  <HelpIcon
-                    className={classnames({ 'kubevirt-form-row__help-icon--hidden': !isActive })}
-                  />
-                }
-              />
-            }
+            bodyContent={help}
           >
-            {help}
-          </PopoverStatus>
-        </span>
-      )}
+            <button
+              type="button"
```

Comment:
do you need type=button?

Reply:
I didn't add it at first, but there's an eslint check that requires buttons to have a type. In my opinion, we shouldn't override those checks without a good reason.

Reply:
ok - I was comparing to what PF has for the implementation and didn't see it. Including shouldn't be harmful in anyway though so it's probably fine.